_scripts/install_all.sh

```diff
@@ -51,6 +53,10 @@ cd fxa-basket-proxy; npm i; cd ..
 
 cd 123done; npm i; cd ..
 
+cd pushbox; docker build -t pushbox .; cd ..
```

Comment:
We should just `docker pull` pushbox instead I think, instead of building locally